recipe/meta.yaml

```diff
@@ -57,11 +58,15 @@ test:
     - Cryptodome.Util
   requires:
     - psutil
+    - pip
+  commands:
+    - pip check
```

Comment:
Could do something like `python -m Cryptodome.SelfTest` here if we're building their test-vectors package.
https://github.com/Legrandin/pycryptodome/blob/dc92e70ffb276d946364f62d0f87c6d66d75ffe3/INSTALL.rst#L88